### DISCUSSION TITLE
What should I implement for lunar spaceflight?

### DISCUSSION THREAD

**Discussion by aarondewindt** [JSBSim-Team/jsbsim#489] [Q&A] [answered]
For my thesis I'm simulating lunar ground to orbit trajectories. For now I've been using a simple 2d 3dof spherical model, but I want to switch to a 3d 6dof model. I've used JSBSim for other projects before, but it's always been Earth atmospheric flight.

So what should I do to simulate lunar spaceflight?
- I guess, new atmospheric model?
- Setting the right gravity, sufrace diameter, etc, on some properties?
- ....?

**Comment by bcoconni** [comment, marked as answer]
You can provide the moon characteristics to JSBSim using the `<planet>` XML element (values based on [NASA's Moon Fact Sheet](https://nssdc.gsfc.nasa.gov/planetary/factsheet/moonfact.html)):
```xml
<planet>
  <semimajor_axis unit="KM">1738.1</semimajor_axis>
  <semiminor_axis unit="KM">1736.0</semiminor_axis>
  <rotation_rate unit="RAD/SEC">2.66167E-6</rotation_rate>
  <GM unit="M3/SEC2">4.9E12</GM>
  <J2>2.027E-4</J2>
</planet>
```
Regarding the Moon atmosphere (or rather the lack of), your best option is to not provide any aerodynamics data. For that, just specify an empty `<aerodynamics>` section:
```xml
<aerodynamics/>
```
